### DISCUSSION TITLE
Will I get memory leak if I never call `unbind()`?

### DISCUSSION THREAD

**Discussion by GF-Huang** [canton7/Stylet#351] [Q&A] [answered]
Is the `PropertyChangedExtensions.Bind` mechanism the same as C# event subscription mechanism?

**Comment by canton7** [comment, marked as answer]
Yes, it subscribes to the PropertyChanged event, so the same rules about unsubscribing apply: you need to unsubscribe if the thing you're subscribing to lives longer than you do
